data-service/src/integration/v1_graphql.entities.test.ts

```diff
@@ -0,0 +1,400 @@
+/**
+ * GraphQL API Integration tests
+ *
+ * @group integration
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const crossFetch = require('cross-fetch');
+
+const url = 'http://localhost:4000/v1/graphql';
```

Comment:
These are still looking at the old API at the moment, until I re-implement in Rust